test/services/edusharing/services/EduSharingConnectorV7.test.js

```diff
@@ -146,6 +150,54 @@ describe('EduSharingV7 FIND', () => {
 			throw new Error(err);
 		}
 	});
+
+	it('player should return h5p data', async () => {
+		const user = await testObjects.createTestUser({ roles: ['teacher'], schoolId: '5fcfb0bc685b9af4d4abf899' });
+		const params = await testObjects.generateRequestParamsFromUser(user);
+		const getStub = sinon.stub(request, 'get');
+
+		getStub.onCall(0).returns(MockH5PRenderer);
+
+		const result = await eduSharingPlayerService.get('dummy-id', params);
+
+		chai.expect(result).to.eql({ iframe_src: 'iframeDummySrc', script_src: 'https://test.test/dummy.js' });
+	});
+
+	it('player should fail when edushare response is invalid', async () => {
+		try {
+			const user = await testObjects.createTestUser({ roles: ['teacher'], schoolId: '5fcfb0bc685b9af4d4abf899' });
+			const params = await testObjects.generateRequestParamsFromUser(user);
+			const getStub = sinon.stub(request, 'get');
+
+			getStub.onCall(0).returns('{}');
+
+			await eduSharingPlayerService.get('dummy-id', params);
+
+			throw new Error('should have failed');
+		} catch (err) {
+			chai.expect(err.message).to.not.equal('should have failed');
+			chai.expect(err.code).to.equal(503);
+			chai.expect(err.message).to.equal('Unexpected response from Edu-Sharing renderer.');
+		}
+	});
+
+	it('player should fail when edushare response has invalid details', async () => {
+		try {
+			const user = await testObjects.createTestUser({ roles: ['teacher'], schoolId: '5fcfb0bc685b9af4d4abf899' });
+			const params = await testObjects.generateRequestParamsFromUser(user);
+			const getStub = sinon.stub(request, 'get');
+
+			getStub.onCall(0).returns('{"detailsSnippet":""}');
+
+			await eduSharingPlayerService.get('dummy-id', params);
+
+			throw new Error('should have failed');
+		} catch (err) {
+			chai.expect(err.message).to.not.equal('should have failed');
```

Comment:
If chai.expect(err.message) equal to "No data detected in Edu-Sharing renderer response." it must be not equal to each other. This line can be removed.

Reply:
Changed in last commit